### ISSUE TITLE
Bottom temperature maps

### ISSUE THREAD

**Issue by khyde** [NOAA-EDAB/ecodata#119]
For the bottom temp maps here: https://noaa-edab.github.io/catalog/bottom-temperature---seasonal-gridded-1.html?
1) Extend the northern range to include all of the GOM
2) Consider a different color scale so that the temperatures are more discernible.  I find it difficult to see the temperature differences in the current maps.
3) I don't know if it belongs here or as a catalog issue, but it would be very helpful to also include bottom temp anomaly maps on this page.

**Comment by BBeltz1** [comment]
@jcaracappa1 any input on this before I make changes?

**Comment by jcaracappa1** [comment]
1. Agree to extend max latitude
2. Agreed. color scale should either be white to red or blue to red
3. We decided to add anomaly plots but not for this year's report

**Comment by BBeltz1** [comment]
I will get on these changes. Thanks.

**Comment by BBeltz1** [comment]
@khyde @jcaracappa1 I've changed the color scale of the bottom temp seasonal gridded maps. Let me know your thoughts.

As for the change to the northern limit, is this something we want to do. It will essentially turn this map into a shelfwide map instead of being separated by report.

Currently, `ecodata::plot_bottom_temp_seasonal_gridded(report = "MidAtlantic")` plots this:
![image](https://github.com/NOAA-EDAB/ecodata/assets/136381970/cf8f3ad8-74cb-4c49-884b-0a7ad49f3726)

And `ecodata::plot_bottom_temp_seasonal_gridded(report = "NewEngland")` plots this:
![image](https://github.com/NOAA-EDAB/ecodata/assets/136381970/50d44ad1-70e7-4ddf-8fd6-cd6ca668f643)

**Comment by jcaracappa1** [comment]
since this isn't going the report, we might just want to plot to be shelf wide with an option for selected one of the EPUs

**Comment by khyde** [comment]
I agree that it should be a shelfwide map and that we don't need EPU specific maps in the catalog.  If anything, I would suggest including the offshore data as well and just make the EPU outlines a little thicker so you can really see the boundary.  We often talk about offshore influences so I think it is important to show that data as well.

For the color scale, I think we need something that is more dynamic.  It is very hard to see the differences in a monochromatic scale.  This is what I am using for the summary pages and I think you can see the different temperatures so much better in something with more colors.
![D_20231008_20231014-ACSPO-V2 81-L3B2-NWA_SUBSET-PXY_1_1211676-SST-STACKED_STATS](https://github.com/NOAA-EDAB/ecodata/assets/5922605/f3a69d93-6a53-464f-af1d-8b5b8c5aba26)

**Comment by jcaracappa1** [comment]
I did not supply gridded bottom temperature for outside the EPUs

**Comment by jcaracappa1** [comment]
I like that scale, but we don't want to confuse it for the color scale used in the gridded anomaly plots

**Comment by khyde** [comment]
True, maybe one that goes from navy to green to yellow?  There are a few here that would work. Maybe Inferno or Viridis https://jmsallan.netlify.app/blog/the-viridis-palettes/

**Comment by BBeltz1** [comment]
yeah I can implement viridis inferno. i'll post new plots when that's done so you can take a look.

**Comment by BBeltz1** [comment]
`ecodata::plot_bottom_temp_seasonal_gridded()`, now in viridus inferno. Thoughts?

![image](https://github.com/NOAA-EDAB/ecodata/assets/136381970/44a3b7c9-483b-4c2a-a98c-c4040b1ca74c)

**Comment by khyde** [comment]
This looks better.  What is the temperature range being plotted?  Maybe extend the min temp to 0 and max at 25 to allow for more dynamics in the lower temps since that is where the majority of the data are.
